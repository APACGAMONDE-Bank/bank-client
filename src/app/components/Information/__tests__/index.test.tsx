import React from 'react';
import { render } from '@testing-library/react';

import { Information } from '..';

describe('<Information  />', () => {
  it('should match snapshot', () => {
    const loadingIndicator = render(<Information />);
    expect(loadingIndicator.container.firstChild).toMatchSnapshot();
  });
});
