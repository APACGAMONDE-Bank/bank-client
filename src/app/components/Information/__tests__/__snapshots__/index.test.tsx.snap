// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Information  /> should match snapshot 1`] = `
.c0 {
  max-width: 1260px;
  margin: 10px auto;
  color: #fff;
  background-color: #1890ff;
  padding: 10px 15px;
  border-radius: 2px;
}

@media (min-width:768px) {
  .c0 {
    padding: 10px 130px;
  }
}

<div
  class="c0"
>
  If you take advantage of our promotion and register your account by the end of this year, you will receive 10,00 USD from us. Accounts already created will also receive this from us.
</div>
`;
